### ISSUE TITLE
Support tts model selection (tts-1, tts-1-1106, tts-1-hd, tts-1-hd-1106)

### ISSUE THREAD

**Issue by mcint** [simonw/ospeak#10]
Support tts model selection to better make use of request request quotas. Support among tts-1, tts-1-1106, tts-1-hd, tts-1-hd-1106. 

For at least me, the -1106 versions support 3k Req/min, versus 3 Req/min for the default.

![Screenshot_20231108_192325](https://github.com/simonw/ospeak/assets/601708/635673cf-b938-4975-9161-6888e7ba29c6)

**Comment by tomviner** [comment]
This would also allow testing out the canary version of tts (`canary-tts`), which was added a few days ago.

As tracked by my OpenAI models API scraper at https://github.com/tomviner/scrape-ai/blame/main/openai/api/models.txt#L14

**Comment by simonw** [comment]
Fixed in #5

**Comment by simonw** [comment]
This works:
```bash
ospeak -m tts-1-hd 'hello' 
```
This shows an error:
```bash
ospeak -m tts-1-hd2 'hello'
```
```
Error: The model `tts-1-hd2` does not exist
```
